tck/util/src/main/java/ee/jakarta/tck/faces/test/util/selenium/ChromeDevtoolsDriver.java

```diff
@@ -296,7 +296,7 @@ public String getPageText() {
     public String getPageTextReduced() {
         String head = delegate.findElement(By.tagName("head")).getAttribute("innerText");
         String body = delegate.findElement(By.tagName("body")).getAttribute("innerText");
-        return (head + " " + body).replaceAll("[\\s\\n ]+", " ");
+        return (head + " " + body).replaceAll("[\\t\\s\\r\\n ]+", " ");
```

Comment:
I'm confused. \s already covers at least \t \r \n and ' '. In fact replaceAll("\\s+", " ") had been sufficient.

Reply:
ok yes you are right the \s covers that part (my fault), nevertheless, there is an npbsp encoding in the regexp which is stripped, this might be the reason if not, then please check the string affected on your machine and give me the character code. I will change the code back! This r an is overkill!

Reply:
Non-breaking space (xA0) is also covered by `\s`.

Reply:
\u00A0 unfortunately not!